app/models/atmosphere/action.rb

```diff
@@ -0,0 +1,24 @@
+module Atmosphere
+  class Action < ActiveRecord::Base
+
+    belongs_to :appliance,
+               class_name: 'Atmosphere::Appliance'
+
+    has_many :action_logs,
+             dependent: :destroy,
+             class_name: 'Atmosphere::ActionLog'
+
+    def log(message, level = :info)
```

Comment:
Unused method argument - `message`. If it's necessary, use `_` or `_message` as an argument name to indicate that it won't be used.